### PR TITLE
chore: Updated hex color in price chart to use ds tokens

### PR DESCRIPTION
**Pull request by brianacnguyen**
## **Description**
Updated hex color in price chart to use ds tokens

## **Related issues**

Fixes: N/A

## **Manual testing steps**

1. Go to wallet
2. Click on a token
3. Click on a point on the chart

### **Before**
https://github.com/MetaMask/metamask-mobile/assets/14355083/05ea057f-c576-40b0-941c-7fef3db662c9

### **After**
https://github.com/MetaMask/metamask-mobile/assets/14355083/76da7977-878c-4717-ad0b-50e19e64cf57

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained what problem this PR is solving and how it is solved.
- [x] I've linked related issues
- [x] I've included manual testing steps
- [x] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [x] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: app/components/UI/AssetOverview/PriceChart/PriceChart.styles.tsx [hunk 1]
```diff
@@ -37,6 +37,12 @@ const styleSheet = (params: { theme: Theme }) => {
     noDataOverlayText: {
       textAlign: 'center',
     } as TextStyle,
+    tooltipLine: {
+      color: theme.colors.icon.alternative,
+    },
+    tooltipCircle: {
+      color: theme.colors.primary.inverse,
```
Comment (georgewrmarshall) [inline app/components/UI/AssetOverview/PriceChart/PriceChart.styles.tsx:44]: I'm trying to figure out which part of the circle this is for if it's the stroke should it not be `success.inverse`? Seeing as it's green?

Screenshot 2024-01-16 at 11 05 04 AM

**Review by georgewrmarshall — COMMENTED** [review]
Looking good! Left a comment

File: app/components/UI/AssetOverview/PriceChart/PriceChart.styles.tsx (hunk 1 above)
Comment (brianacnguyen) [reply app/components/UI/AssetOverview/PriceChart/PriceChart.styles.tsx:44]: its the white circle behind the green

Comment (brianacnguyen) [reply app/components/UI/AssetOverview/PriceChart/PriceChart.styles.tsx:44]: This call was made by @Akatori-Design https://docs.google.com/spreadsheets/d/1kdf7wDNoqmScQAedRn9jE42L-HY0gPqSKEpPd-HGn5A/edit#gid=0

Comment (brianacnguyen) [reply app/components/UI/AssetOverview/PriceChart/PriceChart.styles.tsx:44]: and the chart can also be red, so using success inverse does not make sense in that case

**Review by georgewrmarshall — APPROVED** [review]

**Pull request merged**